cli/cmd/cluster-apply.go

```diff
@@ -107,6 +108,10 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if err := util.PatchServiceAccount("default", kubeconfigPath); err != nil {
```

Comment:
I assume this is an idempotent operation, that is, if the SA is already patched it won't fail. Have you checked that?

Reply:
Yes, it's idempotent, it won't fail if it's already patched. Tested it

Reply:
Willl add a test just for a single component to check it